app/assets/javascripts/admin/addon/templates/components/admin-watched-word.hbs

```diff
@@ -7,3 +7,6 @@
     <span class="tag">{{tag}}</span>
   {{/each}}
 {{/if}}
+{{#if this.isCaseSensitive}}
+  <span class="case-sensitive">({{i18n "admin.watched_words.case_sensitive"}})</span>
```

Comment:
Let's move the parenthesis in the translation so that other locale can use different characters/formatting if needed

```suggestion
  <span class="case-sensitive">{{i18n "admin.watched_words.case_sensitive"}}</span>
```